cmake/freebsd/toolchain-x86_64.cmake

```diff
@@ -10,7 +10,7 @@ set (CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)  # disable linkage check - it
 set (CMAKE_AR "/usr/bin/ar" CACHE FILEPATH "" FORCE)
 set (CMAKE_RANLIB "/usr/bin/ranlib" CACHE FILEPATH "" FORCE)
 
-set (LINKER_NAME "lld" CACHE STRING "" FORCE)
+set (LINKER_NAME "ld.lld" CACHE STRING "" FORCE)
```

Comment:
copied changes from https://github.com/ClickHouse/ClickHouse/pull/20495/files that are about linker

because of error I saw:
```
docker run --network=host --rm --volume=/tmp/build_output_folder:/output --volume=/home/semin-serg/ClickHouse:/build --volume=/tmp/local_ccache:/ccache  -e DEB_CC=clang-13 -e DEB_CXX=clang++-13 -e CC=clang-13 -e CXX=clang++-13 -e CCACHE_DIR=/ccache -e CCACHE_BASEDIR=/build -e CCACHE_NOHASHDIR=true -e CCACHE_COMPILERCHECK=content -e CCACHE_MAXSIZE=15G -e VERSION_STRING='21.10.1.8011' -e AUTHOR='clickhouse' -e CMAKE_FLAGS="$CMAKE_FLAGS -DADD_GDB_INDEX_FOR_GOLD=1 -DLINKER_NAME=lld"  yandex/clickhouse-deb-builder:latest

2021-09-07 20:32:08 Run Build Command(s):/usr/bin/ninja cmTC_5f447 && [1/2] Building CXX object CMakeFiles/cmTC_5f447.dir/src.cxx.o
2021-09-07 20:32:08 clang: warning: argument unused during compilation: '-specs=/usr/share/dpkg/no-pie-compile.specs' [-Wunused-command-line-argument]
2021-09-07 20:32:08 [2/2] Linking CXX executable cmTC_5f447
2021-09-07 20:32:08 FAILED: cmTC_5f447 
2021-09-07 20:32:08 : && /usr/bin/clang++-13  -g -O2 -fdebug-prefix-map=/build=. -specs=/usr/share/dpkg/no-pie-compile.specs -Wdate-time 
-fdiagnostics-color=always -DHAVE_SSSE3 -mssse3 -O0  -Wl,-Bsymbolic-functions -specs=/usr/share/dpkg/no-pie-link.specs --ld-path=lld -rdynamic 
-Wl,--gdb-index -Wl,--build-id=sha1   -rdynamic CMakeFiles/cmTC_5f447.dir/src.cxx.o  -o cmTC_5f447   && :
2021-09-07 20:32:08 clang: warning: argument unused during compilation: '-specs=/usr/share/dpkg/no-pie-compile.specs' [-Wunused-command-line-argument]
2021-09-07 20:32:08 clang: warning: argument unused during compilation: '-specs=/usr/share/dpkg/no-pie-link.specs' [-Wunused-command-line-argument]
2021-09-07 20:32:08 lld is a generic driver.
2021-09-07 20:32:08 Invoke ld.lld (Unix), ld64.lld (macOS), lld-link (Windows), wasm-ld (WebAssembly) instead
2021-09-07 20:32:08 clang: error: linker command failed with exit code 1 (use -v to see invocation)
2021-09-07 20:32:08 ninja: build stopped: subcommand failed.
2021-09-07 20:32:08 
2021-09-07 20:32:08 
2021-09-07 20:32:08 Source file was:
2021-09-07 20:32:08 
2021-09-07 20:32:08         #include <tmmintrin.h>
2021-09-07 20:32:08         int main() {
2021-09-07 20:32:08             __m64 a = _mm_abs_pi8(__m64());
2021-09-07 20:32:08             (void)a;
2021-09-07 20:32:08             return 0;
2021-09-07 20:32:08         }
2021-09-07 20:32:08     
2021-09-07 20:32:08 Performing C++ SOURCE FILE Test HAVE_SSE41 failed with the following output:
2021-09-07 20:32:08 Change Dir: /build/obj-x86_64-linux-gnu/CMakeFiles/CMakeTmp
2021-09-07 20:32:08
```